_includes/manuals/1.12/3.5.2_configuration_options.md

```diff
@@ -385,7 +385,13 @@ See also: dns_conflict_timeout
 
 ##### remote_addr
 
-If Foreman is running behind Passenger or a remote load balancer, the IP of this load balance should be set here. This is a regular expression, so it can support several load balancers, i.e: (10.0.0.1|127.0.0.1)
+When foreman receives client web requests via a smart proxy, proxy or load balancer, the original client source IP address is lost, replaced by the smart proxy, proxy, or load balancers IP instead.  For template requests without a token, this causes a failure, because foreman can't match the request against a valid host.
+
+Smart proxies, and other devices if configured, can preserve the original client IP within an HTTP X-Forwarded-For header, which foreman can evaluate and use to match the request against a valid host.
```

Comment:
Use backticks (`) around X-Forwarded-For so it's clear that it's a literal name, it'll format it correctly.